junifer/datagrabber/datalad_base.py

```diff
@@ -176,7 +176,16 @@ def _dataset_get(self, out: Dict) -> Dict:
             The unmodified input dictionary.
 
         """
-        to_get = [v["path"] for v in out.values() if "path" in v]
+        to_get = []
+        for type_val in out.values():
+            # Iterate to check for nested "types" like mask
+            for k, v in type_val.items():
+                # Add base data type path
+                if k == "path":
+                    to_get.append(v)
+                # Add nested data type path
+                if isinstance(v, dict) and "path" in v:
```

Comment:
Ok. Only two levels supported for now.

Reply:
Yeah I don't see a case of going further in a while (hopefully wouldn't need to).